src/components/core/Form/Form.module.scss

```diff
@@ -0,0 +1,21 @@
+@import 'src/styles/settings.scss';
+
+.c-form {
+  padding: $space-xl $space-3xl;
+  border: 1px solid $color-gray-50;
+  box-shadow: $shadow-lg;
+  border-radius: $border-radius-md;
+  font-size: $fs-base;
+  display: flex;
+  flex-direction: column;
+  gap: $space;
+
+  &__title {
+    font-weight: $fw-bold;
+    font-size: 4.5rem;
+    text-align: center;
+    margin: $space-m 0;
+    font-style: italic;
+    letter-spacing: -1.2px;
```

Comment:
Same as above, please do not use negative and fixed pixel values. Use variables for font sizes